### ISSUE TITLE
ENOENT Error and Path issue?

### ISSUE THREAD

**Issue by BlackBagels** [HashLips/hashlips_art_engine#325]
i keep getting an ENOENT error :

 errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: '/Users/username/Desktop/hashlips_art_engine-1.1.2_patch_v1/layers/Eyeball/'

**Comment by BlackBagels** [comment]
I ended up figuring this out and it was a total error on my part. What I was able to deduce, was that I had not hit save after replace my files. Hope this helps someone else.

**Comment by leroyjware** [comment]
Seeing the same issue. My files are fine.

**Comment by BlackBagels** [comment]
> 

Yeah - I ended up figuring this out and it was a total error on my part. What I was able to deduce, was that I had not hit save after replace my files. Hope this helps someone else.

Once I hit save and then ran node index.js , it worked fine.

**Comment by andyq1995** [comment]
you saved me thanks! i was trying for like 4 hours to figure out the problem, it worked out after saving

**Comment by errahali42** [comment]
Hello friends, I want to develop a project on Solana network by Magic Elaine.I have a team of five people and we have no problem with any field of marketing, but we have a problem with Smart Contract.Can anyone help us in setting up our smart contract on Solana network, especially Magic Eden?
